src/languages/php.js

```diff
@@ -368,6 +353,20 @@ export default function(hljs) {
           2: "variable.language",
         },
       },
+      {
+        match: [
+          PASCAL_CASE_CLASS_NAME_RE,
+          regex.concat(
+            /::/,
+            regex.lookahead(/(?!class\b)/)
```

Comment:
Can you think of a nice short name for `negative lookahead`?  Really this doesn't need the wrapper, and it's being double wrapped now:

```
/(?=(?!class\b))/
```

I'm not asking you to change it, just if you have any ideas... this comes up a lot and we don't currently have a super elegant way to handling the negative case... Perhaps `lookaheadNot`?

```js
regex.lookaheadNot(/class\b/);
```

Reply:
Why just not `regex.negativeLookahead(/(class\b)/)`? After all, that's its name.

Reply:
maybe.